gtsfm/frontend/verifier/ransac.py

```diff
@@ -23,29 +23,32 @@
 from gtsfm.common.keypoints import Keypoints
 from gtsfm.frontend.verifier.verifier_base import VerifierBase, NUM_MATCHES_REQ_E_MATRIX, NUM_MATCHES_REQ_F_MATRIX
 
-PIXEL_COORD_RANSAC_THRESH = 4  # TODO: hyperparameter to tune
-DEFAULT_RANSAC_SUCCESS_PROB = 0.9999
+
+DEFAULT_RANSAC_SUCCESS_PROB = 0.99999
+DEFAULT_RANSAC_MAX_ITERS = 20000
+MAX_TOLERATED_POLLUTION_INLIER_RATIO_EST_MODEL = 0.1
```

Comment:
Same comment as in ransac

Reply:
should be fixed now, I renamed it to `min_allowed_inlier_ratio_est_model` in the config.